### PR TITLE
CB-68: Add swagger

### PR DESCRIPTION
**Pull request by mirecki**
https://team-1626248039387.atlassian.net/jira/software/projects/CB/boards/1?selectedIssue=CB-68

### CODE REVIEW

File: spec/integration/api/v1/recipes_spec.rb [hunk 1]
```diff
@@ -0,0 +1,124 @@
+require 'swagger_helper'
+RSpec.describe 'Recipes API', :request do
+  let(:recipe) { create(:recipe) }
+
+  path '/api/v1/recipes' do
+    get 'List Recipes' do
```
Comment (h4b00) [inline spec/integration/api/v1/recipes_spec.rb:6]: Returns list of recipes

File: spec/integration/api/v1/recipes_spec.rb [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+      }
+      response '200', 'create and returns recipe' do
+        let(:recipe) { create(:recipe) }
+
+        run_test!
+      end
+    end
+  end
+  path '/api/v1/recipes/{id}' do
+    get 'shows the Recipe' do
```
Comment (h4b00) [inline spec/integration/api/v1/recipes_spec.rb:40]:
```suggestion
    get 'Returns recipe' do
```

**Review by h4b00 — APPROVED** [review]
You are almost there, few tweaks and you can merge.

File: swagger/v1/swagger.json [hunk 3]
```diff
@@ -0,0 +1,307 @@
...
+              "application/json": {
+                "data": [
+                  {
+                    "id": 3164,
+                    "name": "Barbecue Ribs",
+                    "content": "28-day aged 300g USDA Certified Prime Ribeye, rosemary-thyme garlic butter, with choice of two sides.",
+                    "video_link": "https://ebert.io/johnie_schneider",
+                    "cooking_time": 55
+                  },
+                  {
```
Comment (grabomarcin) [inline swagger/v1/swagger.json:27]: Why don't you return the price in recipe?

**Review by grabomarcin — APPROVED** [review]

**Review by Narzew — APPROVED** [review]

**Pull request merged**